collection/get-geographical-location.yml

```diff
@@ -0,0 +1,33 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: get geographical location
+    namespace: collection
+    author: moritz.raabe
+    scope: function
+    att&ck:
+      - Discovery::System Location Discovery [T1614]
+    examples:
+      - 9879D201DC5ACA863F357184CD1F170E:0x10001A99
+  features:
+    - or:
+      - api: GetLocaleInfo
+      - api: GetLocaleInfoEx
+      - api: GetKeyboardLayout
+      - api: GetKeyboardLayoutList
+      - api: GetKeyboardLayoutName
```

Comment:
seems like this might be a little loose. can you convince me?

Reply:
I think keyboard layout is not the same as location (I am using US-layout but I am not in the USA). Maybe it makes sense to split this in a different rule.

Reply:
updated this rule, is this better?